app/models/concerns/wp_post.rb

```diff
@@ -12,7 +12,8 @@ def update_post(json)
     end
 
     self.wp_id        = json['ID']
-    self.published_at = json['date']
+    # Use gmt date to ignore timezone settings in WordPress
+    self.published_at = json['date_gmt']
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.